### PR TITLE
Add ibis-omniscidb recipe

### PR DESCRIPTION
**Pull request by xmnlab**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by xmnlab** [comment]
ibis-omniscidb license was already included to the repository(https://github.com/omnisci/ibis-omniscidb/blob/master/LICENSE.txt), and it would be available for the next version.

**Comment by xmnlab** [comment]
The current version of ibis-framework (1.4) has still support for omniscidb (just for python 3.7), but this support will be dropped in the next version (2.0).

This recipe, for now, will support just python 3.8 and python 3.9

File: recipes/ibis-omniscidb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  sha256: 3b85846e31d3eb6d4fa3ed1236fc205e2fb2029a69b6c8c9a3746b2bcdc5076b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (beckermr) [inline recipes/ibis-omniscidb/meta.yaml:19]: You need a min version here too.

**Review by beckermr — CHANGES_REQUESTED** [review]

File: recipes/ibis-omniscidb/meta.yaml (hunk 1 above)
Comment (xmnlab) [reply recipes/ibis-omniscidb/meta.yaml]: done! thank you so much for the review.

**Review by beckermr — APPROVED** [review]

**Comment by xmnlab** [comment]
thank you so much @beckermr

**Pull request merged**